src/custom/ShareModal/style.tsx

```diff
@@ -0,0 +1,70 @@
+import { styled } from '@mui/material';
+import { DialogContentText, FormControl, ListItemText, Select, Typography } from '../../base';
+
+export const CustomText = styled(Typography)(() => ({
+  fontFamily: 'Qanelas Soft, sans-serif',
```

Comment:
you don't have to pass this , at global level font family  is this only .

Reply:
Okay